projects/convai2/baselines/seq2seq/eval_f1.py

```diff
@@ -22,6 +22,8 @@
     )
     opt = parser.parse_args(print_args=False)
     opt['model_type'] = 'seq2seq'
-    fnames = ['convai2_self_seq2seq_model.tgz', 'dict_convai2_self']
-    download_models(opt, fnames, 'convai2')
+    opt['override'] = ['dict_file', 'dict_lower', 'batchsize']
```

Comment:
oh is this the same prob we talked about before? we should just fix it by replacing  "set_defaults" function as we said?